### ISSUE TITLE
3.1.0 Simply don't launch

### ISSUE THREAD

**Issue by MmeCromatique** [pascallj/mineshot-revived-fabric#12]
I've just dicovered this mod and installed it, my game refused tu launch wth an error code -1, I deleted all of my other mods exept ModMenu and Fabric API and still nothing, but using 3.0.0 works and with all my other mods
```
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_761 failed
```
Crash Report:
https://pastebin.com/dUVLV5uj

**Comment by pascallj** [comment]
Thank you for your report! Apparently they changed some functions we use between 1.18.1 and 1.18.2. I was still testing on 1.18.1.

Fixed in 77c54acd4572923a3b54f4b8dcc82353c97066c5 and also released as [v3.1.1](https://github.com/pascallj/mineshot-revived-fabric/releases/tag/v3.1.1).

Please let me know when you still have issues.

**Comment by pascallj** [comment]
For the sake of it, I tried it in a non-development environment and it still doesn't work. Apparently the development handles the remapping differently, which is why in testing it always worked for me.

Reopening this for now, will have a look at it later.

**Comment by pascallj** [comment]
Couldn't resist fixing this bug right away. Should all be good now.

Fixed in bd2493b and also released as [v3.1.2](https://github.com/pascallj/mineshot-revived-fabric/releases/tag/v3.1.2).
